shared_model/validators/field_validator.cpp

```diff
@@ -11,6 +11,7 @@
 #include <boost/algorithm/string_regex.hpp>
 #include <boost/format.hpp>
 #include <boost/range/adaptor/indexed.hpp>
+#include <string_view>
```

Comment:
Looks a bit out of place, but probably OK since it is not picked up by clang-format.

Reply:
it is not ok, thanks